1.2/Dockerfile

```diff
@@ -91,19 +98,23 @@ RUN mkdir -p /opt/npm/2.15.8/node_modules \
   && rm -f node-v6.9.1-linux-x64.tar.xz \
   && mv /opt/nodejs/node-v6.9.1-linux-x64 /opt/nodejs/6.9.1 \
   && echo "3.10.8" > /opt/nodejs/6.9.1/npm.txt \
+  && tar xfJ /tmp/node-v6.9.3-linux-x64.tar.xz -C /opt/nodejs \
+  && rm -f node-v6.9.3-linux-x64.tar.xz \
+  && mv /opt/nodejs/node-v6.9.3-linux-x64 /opt/nodejs/6.9.3 \
+  && echo "3.10.10" > /opt/nodejs/6.9.3/npm.txt \
   && chown -R root:root /opt/nodejs \
-  && ln -s /opt/nodejs/6.9.1/bin/node /opt/nodejs/node \
-  && ln -s /opt/nodejs/6.9.1/npm.txt /opt/nodejs/npm.txt \
+  && ln -s /opt/nodejs/6.9.3/bin/node /opt/nodejs/node \
+  && ln -s /opt/nodejs/6.9.3/npm.txt /opt/nodejs/npm.txt \
   && rm -f /usr/bin/node \
-  && ln -s /opt/nodejs/6.9.1/bin/node /usr/bin/node \
-  && rm -f /opt/nodejs/6.9.1/bin/npm \
-  && ln -s /opt/npm/3.10.8/node_modules/npm/bin/npm /opt/nodejs/npm \
-  && ln -s /opt/npm/3.10.8/node_modules /opt/nodejs/node_modules \
+  && ln -s /opt/nodejs/6.9.3/bin/node /usr/bin/node \
+  && rm -f /opt/nodejs/6.9.3/bin/npm \
+  && ln -s /opt/npm/3.10.10/node_modules/npm/bin/npm /opt/nodejs/npm \
+  && ln -s /opt/npm/3.10.10/node_modules /opt/nodejs/node_modules \
   && rm -rf /usr/bin/npm \
-  && ln -s /opt/npm/3.10.8/node_modules/npm/bin/npm /usr/bin/npm \
-  && ln -s /opt/npm/3.10.8/node_modules /usr/bin/node_modules
+  && ln -s /opt/npm/3.10.10/node_modules/npm/bin/npm /usr/bin/npm \
+  && ln -s /opt/npm/3.10.10/node_modules /usr/bin/node_modules
   
-ENV PATH $PATH:/opt/nodejs/6.9.1/bin
+ENV PATH $PATH:/opt/nodejs/6.9.3/bin
```

Comment:
I'm not sure whether this is actually needed since Node.js 6.9.3 is actually symlinked into `/usr/bin` which should make it the default.